packages/cli/src/commands/service/dev.ts

```diff
@@ -1,87 +1,112 @@
-import { IService } from '@mesg/api/lib/service'
-import * as base58 from '@mesg/api/lib/util/base58'
+import { Command, flags } from '@oclif/command'
+import { join } from 'path'
+import Listr from 'listr'
+import LCD from '@mesg/api/lib/lcd'
+import API from '@mesg/api'
+import chalk from 'chalk'
+import { decode } from '@mesg/api/lib/util/encoder'
+import { parseLog } from '../../utils/docker'
+import * as Environment from '../../tasks/environment'
+import * as Service from '../../tasks/service'
+import * as Runner from '../../tasks/runner'
+import * as Instance from '../../tasks/instance'
+import version from '../../version'
 
-import Command from '../../root-command'
-import { IsAlreadyExistsError } from '../../utils/error'
+const ipfsClient = require('ipfs-http-client')
 
-import ServiceCompile from './compile'
-import ServiceCreate from './create'
-import ServiceLog from './logs'
-import ServiceStart from './start'
-import ServiceStop from './stop'
+type Context = Environment.IStop | Service.ICompile | Service.ICreate | Runner.ICreate | Runner.ILogs | Instance.IEventLogs | Runner.IResultLogs
+type ContextEnd = Runner.ILogsStop | Instance.IEventLogsStop | Runner.IResultLogsStop | Environment.IStop | Environment.IClearConfig
 
-export default class ServiceDev extends Command {
-  static description = 'Run a service in development mode'
+export default class Dev extends Command {
+  static description = 'Run a service in a local development environment'
 
   static flags = {
-    ...Command.flags,
-    ...ServiceCreate.flags,
-    ...ServiceStart.flags,
+    image: flags.string({ name: 'MESG engine image', default: 'mesg/engine' }),
```

Comment:
let's remove the useless "MESG"
```suggestion
    image: flags.string({ name: 'Engine image', default: 'mesg/engine' }),
```